website/versioned_docs/version-3.4.0/concepts-shops.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Shops
+id: version-3.4.0-concepts-shops
+original_id: concepts-shops
+---
+
+The Reaction Commerce API is a **multi-shop system**, which means that more than one shop can exist within the same installation. Most API calls require you to specify a shop or shops. But what exactly is a shop?
+
+While you might think you intuitively know what a “shop” is, things can become fuzzy when there are complex scenarios like multiple frontend apps, or shops that sell items from other shops. In fact, it is really up to you to decide what you want a “shop” to be in your system. But there are also some rules you have to know to make this decision wisely.
+
```

Comment:
Should we include a note about multi-shop UI support out of the box?
Perhaps something like?

> While the Reaction Commerce API has multi-shop concepts built in, the default UIs for Reaction Commerce (`example-storefront`, `reaction-admin`) have been built to support a single-shop setup out of the box. Any multi-shop UI needs will require additional development work.